service-front-v2/app/templates/component.html

```diff
@@ -0,0 +1,141 @@
+{% extends "base.html" %}
```

Comment:
This looks like it's been committed in error.

Reply:
Agreed. now fixed.